ruby/vscode/minitest/runner.rb

```diff
@@ -25,20 +25,53 @@ def add_dir(path)
 
       def add_file(file)
         test = VSCode::Minitest.tests.find_by(full_path: file)
-        runnables << [test[:klass].constantize, {}] if test
+        runnables << [constant_for(test[:klass]), {}] if test
       end
 
       def add_file_with_line(file, line)
         test = VSCode::Minitest.tests.find_by(full_path: file, line_number: line)
         raise "There is no test the the given location: #{file}:#{line}" unless test
-        runnables << [test[:klass].constantize, filter: test[:method]]
+        runnables << [constant_for(test[:klass]), filter: test[:method]]
       end
 
       def run
         runnables.each do |runnable, options|
           runnable.run(reporter, options)
         end
       end
+
+      private
+
+      def constant_for(str)
```

Comment:
We should have a comment to indicate where this code comes from:

```suggestion
      # Taken from https://github.com/rails/rails/blob/fb3ecbf130e141688782bafb7da979bf0ecedf1a/activesupport/lib/active_support/inflector/methods.rb#L271
      def constant_for(str)
```

Reply:
Great idea, thanks. I'll make that change shortly.

Reply:
Okay, that was easier than I realized to apply the suggestion. Thanks!